packages/browser-ui/src/inspector/views/GraphUtils.ts

```diff
@@ -0,0 +1,258 @@
+import {
+  ShapeTypes,
+  PenroseState,
+  PenroseFn,
+  prettyPrintFn,
+  prettyPrintPath,
+  prettyPrintExpr,
+} from "@penrose/core";
+import { FieldDict, Translation } from "@penrose/core/build/dist/types/value";
```

Comment:
Are these types not exported by `@penrose/core`? Not related to this PR: we should figure out how to export things in general. Seems like we are already having too many top-level exports. Maybe something like `import { FieldDict, Translation } from @penrose/core/style`? cc: @maxkrieger

Reply:
No, they are not. This import is what VSCode put in as a "quick fix." Anyway, I fixed it by exporting from core.